### PR TITLE
Indented lists

### PR DESCRIPTION
**Pull request by sjcross**

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -48,11 +48,21 @@ h3 {
 
 /* List defaults */
 ol {
-  @apply list-inside list-decimal;
+  @apply list-inside list-decimal pl-6 pb-2;
+  > li > ul {
+    list-style-type: circle;
+  }
```
[the remark this answers is not in the document]
Comment (jdjfisher) [reply src/app/globals.css:54]: Perhaps something like the following

```css
ul, ol {
  @apply list-inside pl-6 pb-2;
}

ul {
  @apply list-disc;
}

ol {
  @apply list-decimal;
}

li > ul, ol {
  @apply list-[circle];
}
```

Also just double checking did you try `list-outside` with the `pl-6`? I think that might look a tad better.

**Review by jdjfisher — COMMENTED** [review]
Look good just a small query about a possible issue with the css nesting.

File: src/app/globals.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (sjcross) [reply src/app/globals.css]: Looks good; I've switched these over now.  Only difference is I increased pl-8 to pl-8 as it was beginning to look a bit crowded with list-outside.

File: src/components/TreeNav.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default function TreeNav({ node, depth = 1 }: Props) {
   );
 
   return (
-    <ul className="list-none text-gray-500">
+    <ul className="tree-item list-none text-gray-500">
       {internalChildren.map((child) => (
```
[the remark this answers is not in the document]
Comment (sjcross) [reply src/components/TreeNav.tsx:23]: Also switched this over

**Review by jdjfisher — APPROVED** [review]
Nice work 💯

**Pull request merged**